docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -14,6 +14,28 @@ In the `package.json` file, change the package version from `latest` to `next`.
 Using `next` ensures your project always uses the latest v7 pre-releases.
 Alternatively, you can also target and fix it to a specific version, for example, `7.0.0-alpha.0`.
 
+## Supported browsers and versions
+
+### Minimum TypeScript version
+
+The minimum supported version of TypeScript has been increased from v4.7 to 4.9.
+
+:::info
+We align with [support window](https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#support-window) by [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) (published on npm under the `@types` namespace).
+
+We will not change the minimum supported version in a minor version of Material UI.
+However, we recommend not using a TypeScript version older than the lowest supported version by DefinitelyTyped.
+:::
+
+If your project includes these packages, you'll need to update them:
```

Comment:
If possible, I'd specify the minimum required version

Reply:
Added the section above (I lean on using the same structure as v6 upgrade guide).

Reply:
Do we know what `@types/react` and `@types/react-dom` is the minimum that will work? If we do, I think we can add it to make it easier for users.